docs/api.md

```diff
@@ -147,11 +147,10 @@ Puppeteer is a Node library which provides a high-level API to control Chromium
 
 ### Environment Variables
 
-Puppeteer looks for certain environment variables to aid its operations:
+Puppeteer looks for certain [environment variables](https://en.wikipedia.org/wiki/Environment_variable#Assignment) to aid its operations:
```

Comment:
nit: let's have a link just to wikipedia article

Reply:
done, BTW what is nit?